qt/widgets/instrumentview/src/InstrumentWidgetMaskTab.cpp

```diff
@@ -575,6 +574,19 @@ void InstrumentWidgetMaskTab::singlePixelPicked(size_t pickID) {
   m_instrWidget->updateInstrumentDetectors();
   QApplication::restoreOverrideCursor();
   enableApplyButtons();
+
+  if (m_grouping_on->isChecked()) {
+    if (m_pixel->isChecked()) {
+      m_instrWidget->updateInfoText(
+          QString("Pixel %0 picked for grouping").arg(pickID));
+
+    } else if (m_tube->isChecked()) {
+      std::string message = "Parent " +
```

Comment:
Can you not use QString here as well as above? Also `Parent` might confuse the user, I think `Component` is more friendly.